src/librustc_session/lint/builtin.rs

```diff
@@ -386,6 +386,12 @@ declare_lint! {
     "failures in resolving intra-doc link targets"
 }
 
+declare_lint! {
+    pub INVALID_CODEBLOCK_TAG,
+    Warn,
+    "codeblock tag looks a lot like a known one"
```

Comment:
Here you call them tags rather than flags.  Which is it?

Reply:
I don't know actually. Any suggestion is very welcome. :)

Reply:
The rustdoc book calls these 'attributes'

https://doc.rust-lang.org/rustdoc/documentation-tests.html#attributes